### PR TITLE
Ft delete question 162677079

### PR DESCRIPTION
**Pull request by KelynPNjeri**

### CODE REVIEW

File: app/api/v1/models/questions_model.py [hunk 1]
```diff
@@ -47,3 +47,13 @@ def edit_single_question(question_id, title, description):
         else:
             return "Question Not Found"
 
+    @staticmethod
+    def delete_question(question_id):
+        """Deals with deleting a single question."""
+        questions = Question.questions_list
+        single_question = [qn for qn in questions if qn["Question Id"] == question_id]
+        if single_question:
+            Question.questions_list.remove(single_question[0])
```
[the remark this answers is not in the document]
Comment (KelynPNjeri) [reply app/api/v1/models/questions_model.py]: Wait, it's all my tests that are not working. Not just the delete one.

[the remark this answers is not in the document]
Comment (khwilo) [reply app/api/v1/models/questions_model.py]: Are the methods working manually when you test on **POSTMAN**?

[the remark this answers is not in the document]
Comment (KelynPNjeri) [reply app/api/v1/models/questions_model.py]: Yes the methods are working as expected

[the remark this answers is not in the document]
Comment (KelynPNjeri) [reply app/api/v1/models/questions_model.py]: For questions, it is `/api/v1/questions`

**Pull request merged**